src/Selenium2Library/keywords/_element.py

```diff
@@ -297,6 +297,16 @@ def get_horizontal_position(self, locator):
             raise AssertionError("Could not determine position for '%s'" % (locator))
         return element.location['x']
 
+    def get_element_size(self, locator):
+        """Returns width and height of element identified by `locator`.
+
+        The element width and height is returned.
+        Fails if a matching element is not found.
```

Comment:
Would be nice if there was an example like with [Get Window Size](http://robotframework.org/Selenium2Library/doc/Selenium2Library.html#Get%20Window%20Size).